doc/platform/app/app_roles.rst

```diff
@@ -238,7 +215,60 @@ This means that all the dependencies of a role should be defined in the ``roles`
 
 You can find the full example here: `application_role_cfg <https://github.com/tarantool/doc/tree/latest/doc/code_snippets/snippets/config/instances.enabled/application_role_cfg>`_.
 
+.. _roles_create_custom_role_on_event_callback:
+
+On_event callback
```

Comment:
BTW, there is the 'Specifics of creating spaces' below and it is written before the `on_event` callback was implemented. We possibly should recommend to use the `on_event` callback for this scenario, because it should be simpler. Reach me or @grafin for examples if needed.